Tribler/Core/Modules/restapi/channels/channels_torrents_endpoint.py

```diff
@@ -61,14 +66,35 @@ def render_GET(self, request):
 
             :statuscode 404: if the specified channel cannot be found.
         """
-        channel_info = self.get_channel_from_db(self.cid)
-        if channel_info is None:
-            return ChannelsTorrentsEndpoint.return_404(request)
+        chant_dirty = False
+        if self.is_chant_channel:
+            with db_session:
+                channel = self.session.lm.mds.ChannelMetadata.get(public_key=self.cid)
+                if channel:
+                    if channel == self.session.lm.mds.get_my_channel():
+                        # That's our channel, it gets special treatment
+                        uncommitted = map(lambda x: convert_torrent_metadata_to_tuple(x, UNCOMMITTED),
```

Comment:
Could you rename `x` to something more descriptive?

Reply:
`convert_torrent_metadata_to_tuple` already tells the reader that the `x` is `torrent_metadata`. If we change it to something longer the line will break and readability will suffer.
Really, long identifiers are only good for long scopes. At small scopes, short identifiers are much more readable.